FirstTask.py

```diff
@@ -0,0 +1,180 @@
+from os import system as sys
+
+class BankAccount:
+    def __init__(self, Name, PassCode, AccNumber, Balance, AccoutType):
+        self.Name = Name
+        self.Passcode = PassCode
+        self.Balance = Balance
+        self.AccountType = AccoutType
+        self.AccNumber = AccNumber
+
+    def FullName(self):
+        return self.Name
+
+    def AccountNumber(self):
+        return self.AccNumber
+
+    def PassCode(self):
+        return self.Passcode
+
+    def CheckBalance(self):
+        return self.Balance
+
+    def CheckAccountType(self):
+        return self.AccountType
+
+    def AccDetails(self):
+        print("Account # ", self.AccNumber)
+        print("Name: ", self.Name)
+        print("Balance: ", self.Balance)
+        print("Account Type: ", self.AccountType)
+
+    def DepositBill(self,Bill):
+        if Bill >= 0:
+            self.Balance += Bill
+            return True
+        else:
+            return False
+
+    def DrawMoney(self,Money):
+        if self.Balance >= Money:
+            self.Balance -= Money
+            return True
+        else:
+            return False
+    
+    def TransferMoney(self,Amount):
+        if self.Balance >= Amount:
+            self.Balance -= Amount
+            return True
+        else:
+            return False

```

Comment:
Name should be `transfer_money`. This method should take `sender_acc_no`, `receiver_acc_no`, and the `amount`. If the sender and receiver exists and the amount is valid then it should be transferred from sender's account to the receiver's account.